README.md

```diff
@@ -1,55 +1,120 @@
-# IntegriTEE Parachain:
+# Integritee Parachain
 
-This is the repository to run integritee as a parachain in the rococo-v1 testnet. It is forked from the [Cumulus](https://github.com/paritytech/cumulus) repository and only adds the SubstraTEE pallet and configuration.
+This is the repository to run Integritee as a parachain on Kusama and Rococo. It is forked from the [Cumulus](https://github.com/paritytech/cumulus) repository.
 
 ## Launch a local setup including a Relay Chain and a Parachain
 
 Quick: see [polkadot-launch](https://github.com/paritytech/polkadot-launch.git)
 ```
 node ../polkadot-launch/dist/cli.js launch-kusama-local-with-shell.json
 ```
+or
+```
+polkadot-launch launch-kusama-local-with-shell.json
+```
+depending on how you installed `polkadot-launch`.
 
+### Manually Launch a local Rococo Testnet
 
-### Manually Launch Local Rococo Testnet
+#### Launch a Rococo Relay Chain
 
+First, check out which polkadot release to use in the .json files in the [polkadot-launch](./polkadot-launch) folder:
```

Comment:
here you are mixing the manual approach with polkadot-launch. Don't do that please! I wonder if it is necessary to describe the manual process anymore as I didn't need that one for ages now. What I needed is manually starting additional collators (i.e. a new binary version) That would be more valuable to explain